rskj-core/src/test/java/co/rsk/peg/P2shErpFederationTest.java

```diff
@@ -40,10 +41,13 @@
 import org.junit.jupiter.params.provider.MethodSource;
 
 class P2shErpFederationTest {
-    private P2shErpFederation federation;
+    public static long MAX_CSV_VALUE = 65535L; // 2^16 - 1, since bitcoin will interpret up to 16 bits as the CSV value
```

Comment:
It'd be preferable to read this value from ErpRedeemScriptBuilderUtils. That way if it ever changes it only needs to change in one place and everything else keeps working as expected